src/addresses.py

```diff
@@ -7,6 +7,7 @@
 from binascii import hexlify, unhexlify
 from struct import pack, unpack
 
+from bmconfigparser import BMConfigParser
```

Comment:
since this adds an import, it may be better to put the functions into `BMConfigParser` directly and make them methods

Reply:
Alright, i'll create a new PR to do update in `BMConfigparser`.